Mathlib/Data/Nat/Basic.lean

```diff
@@ -364,7 +364,10 @@ set_option push_neg.use_distrib true in
 lemma one_lt_mul_iff : 1 < m * n ↔ 0 < m ∧ 0 < n ∧ (1 < m ∨ 1 < n) := by
   constructor <;> intro h
   · contrapose! h; simp_rw [Nat.le_zero] at h
-    obtain rfl | rfl | h := h; simp; simp; exact Nat.mul_le_mul h.1 h.2
+    obtain rfl | rfl | h := h
+    · simp
+    · simp
+    · exact Nat.mul_le_mul h.1 h.2
```

Comment:
Is there a way to fit this in one line? In Lean 3 I'd just do `{simp}, {simp}, {exact ...}` which (I think) corresponds to `· simp; · simp; · exact ...` in Lean 4 but I can't get it work; `obtain rfl | rfl | h := h <;> simp <|> exact Nat.mul_le_mul h.1 h.2` doesn't work either IIRC. Am I missing a trick to combine `simp <|> exact Nat.mul_le_mul h.1 h.2` into a block?